include/SugarObjects/templates/basic/Basic.php

```diff
@@ -82,76 +88,100 @@ public function get_summary_text()
      * @throws RuntimeException
      * @throws InvalidArgumentException
      */
-    public function getEmailAddressConfirmOptInTick($emailField)
+    public function getConfirmOptInTickFromSugarEmailAddressField($emailField)
```

Comment:
Update for php doc: This method doesn't throws `RuntimeException` anymore.

Reply:
i have updated updated php doc.